infra/aws/terraform/prow-build-cluster/resources/monitoring/node-exporter/service-monitor.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: node-exporter
+  namespace: monitoring
+  labels:
+    prometheus: main
+spec:
+  selector:
+    matchLabels:
+      app: node-exporter
+  endpoints:
+    - port: metrics
+      # scrapeTimeout: 30s
```

Comment:
Can you leave some comment why is this commented?

Reply:
That should be actually removed.